.travis.yml

```diff
@@ -10,18 +10,19 @@ git:
 
 addons:
   apt:
+    update: true
     sources:
       - ubuntu-toolchain-r-test
-      - sourceline: 'ppa:kzemek/boost'
+      - sourceline: 'ppa:rexut/factory'
     packages:
       - g++-6
-      - libboost1.58-dev
-      - libboost-filesystem1.58-dev
-      - libboost-iostreams1.58-dev
-      - libboost-program-options1.58-dev
-      - libboost-regex1.58-dev
-      - libboost-system1.58-dev
-      - libboost-thread1.58-dev
+      - libboost1.62-dev
```

Comment:
are these changes still required or can they be reverted ?

Reply:
up to you/all to decide, not required indeed

Reply:
maybe save them somewhere in case we have to upgrade boost but remove these changes from the PR, ty

Reply:
👍

Reply:
https://gist.github.com/ccrs/53d049c0dd8b392b3aa3f3c5de91a04b